### DISCUSSION TITLE
Seeed S2120 - Helium ?

### DISCUSSION THREAD

**Discussion by dsandels** [danak6jq/Seeed-LoRa-E5#24] [General]
Hi -

Does anyone know the App EUI and the App Key for the S2120 to connect to the Helium network ?

Thanks,

Dave

**Comment by danak6jq** [comment]
Looks like a question for Helium to me, but I can summarize with:

- create a Helium 'console', load up some DCs
- add your device, any device, with DevEUI and AppKey
- use up those DCs

**Comment by dsandels** [comment]
Thanks Dana - I found them in the Seeed SenseCap mobile app configuration
page.

On Tue, Jul 25, 2023 at 12:48 PM Dana Myers ***@***.***>
wrote:

> Closed #24 <https://github.com/danak6jq/Seeed-LoRa-E5/discussions/24> as
> resolved.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/danak6jq/Seeed-LoRa-E5/discussions/24>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APPZGZJZ6BERHWCA4MPRRTTXSABHFANCNFSM6AAAAAA2XNFBHU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>


-- 
David Sandel
iPhone 314-435-3658
Fax 314-862-4937
Twitter @dsandel @iNeighborhoods @EDesignsCenter
Founder i*iNeighborhoods <http://ineighborhoods.us/>*
6900 Delmar, St. Louis, Missouri 63130
MapSTL <http://ineighborhoods.us/digitaldivide> - Mapping the Digital Divide
My LinkedIn Profile <http://linkedin.com/in/smartcitymasterplanner>
